service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/PlacementSubmittedControllerTest.java

```diff
@@ -56,18 +57,18 @@
     }
 
     @Test
-    void shouldSendNotificationWhenAddingNewChildPlacement() throws Exception {
+    void shouldNotifyHmctsAdminWhenAddingNewChildPlacementAndCtscIsDisabled() throws Exception {
         Element<Child> child1 = testChild();
         Element<Child> child2 = testChild();
 
-        Element<Placement> child1Placement = element(testPlacement(child1, testDocument()));
-        Element<Placement> child2Placement = element(testPlacement(child2, testDocument()));
+        Element<Placement> child1Placement = element(testPlacement(child1, documentReference));
+        Element<Placement> child2Placement = element(testPlacement(child2, documentReference));
```

Comment:
why change here? should really be different instances of doc, no?

Reply:
Changed to be a constant.